modules/accounts.js

```diff
@@ -110,7 +111,8 @@ Accounts.prototype.getAccounts = function (filter, fields, cb) {
  * @returns {setImmediateCallback} Errors.
  * @returns {function()} Call to logic.account.get().
  */
-Accounts.prototype.setAccountAndGet = function (data, cb) {
+// TODO: Remove this bad boy, replace with only get :D
```

Comment:
setSender -> get or setAccountAndGet -> getSender which is done?

Reply:
setAccountAndGet is now getSender